packages/form-state-manager/src/utils/manager-api.ts

```diff
@@ -25,6 +25,9 @@ import FieldConfig from '../types/field-config';
 import { Meta } from '../types/use-subscription';
 import { formLevelValidator, isPromise } from './validate';
 import { FormValidator, FormLevelError } from '../types/validate';
+import { isEqual } from 'lodash';
```

Comment:
I don't see this used. Maybe I am just blind.

Reply:
yep, the lint does not check unused vars... I could set it up.